comp/core/status/component.go

```diff
@@ -21,6 +21,8 @@ const CollectorSection string = "collector"
 type Component interface {
 	// Returns all the agent status information for the format type
 	GetStatus(format string, verbose bool, excludeSection ...string) ([]byte, error)
+	//
+	GetSections() []string
```

Comment:
Why don't we return a byte array as the rest of the functions? 

We could even accept a format type, that way the called of the function what type of data would be in the byte array and we can do manipulate it accordingly.

in the case of showing in the CLI we would use the text format. We can return a comma separate list of sections and the CLI would split by comma and display each of them

Reply:
At the begining I was returning a `[]byte` to match other methods, as you are suggesting.
To be honest I don't have a strong opinion on it, I'm just going back and forth between `[]string` and `[]byte`.
@pgimalac do you have a strong opinion ?